src/views/unauthenticated/UnauthenticatedHome.tsx

```diff
@@ -7,30 +8,34 @@ export function UnauthenticatedHome() {
 	const navigate = useNavigate();
 
 	return (
-		<div className="Home">
-			<h2>Welcome to GrocerEase</h2>
-			<p>
-				The next best thing to having someone else do the shopping for you!
+		<div className="home">
+			<h2 className="heading-text">Welcome to GrocerEase</h2>
+			<p className="normal-text">
+				The next best thing to having someone else do the shopping for you!{" "}
+				<br />
 				Create and manage smart lists, while it learns your shopping habits to
 				let you know exactly when you will need to buy that next batch of toilet
 				paper!
 			</p>
 
 			<article>
-				<p>New to GrocerEase:</p>
-				<Button
-					onClick={() => navigate("/about")}
-					aria-label="Navigate to the about application page."
-					className="m-2"
-				>
-					Learn More
-				</Button>
-				<SignInButton isSignIn={false} />
+				<h3 className="heading-text">Welcome Back:</h3>
```

Comment:
I love the seperation of the buttons here and switching the learn more to the below the sign back in button!